nuxt-frontend/pages/services.vue

```diff
@@ -5,7 +5,7 @@
       <LandingSection />
       <WhatWeOfferMobile class="block lg:hidden" />
       <WhatWeOfferDesktop class="hidden lg:block" />
-      <CTASection class="!mt-24 xl:!mt-40" />
+      <CTASection />
       <TechnologyStack />
       <BlogSection ref="blogs" />
```

Comment:
have you fixed that blog image issue?

Reply:
yes , you can just check it by removing all cache its working